presto-native-execution/CMakeLists.txt

```diff
@@ -171,6 +171,7 @@ find_library(WANGLE wangle)
 
 find_library(RE2 re2)
 
+find_package(fizz CONFIG)
```

Comment:
this is not ideal, but the way to fix it should be to replace all find_library()s by find_package()s, as per our cmake expert @assignUser :)